jepsen/src/jepsen/net.clj

```diff
@@ -46,75 +48,118 @@
 
 (def tc "/sbin/tc")
 
+(defn net-dev
+  "Returns the network interface of the current host."
+  []
+  (let [choices (su (exec :ls "/sys/class/net/"))
+        iface (->> choices
+                   (str/split-lines)
+                   (remove (fn [iface] (re-find #"^lo" iface)))
+                   first)]
+    (assert iface
+            (str "Couldn't determine network interface!\n" choices))
+    iface))
+
+(defn qdisc-del
+  "Deletes root qdisc for given dev on current node."
+  [dev]
+  (try+
+   (su (exec tc :qdisc :del :dev dev :root))
+   (catch [:exit 2] {}
```

Comment:
Is this {} meant to be _? Usually it'd be a binding form there, right?

Reply:
yes, good catch.